pkgs/applications/science/misc/root/default.nix

```diff
@@ -200,6 +204,6 @@ stdenv.mkDerivation rec {
 
     # See https://github.com/NixOS/nixpkgs/pull/192581#issuecomment-1256860426
     # for some context on issues on aarch64.
-    broken = stdenv.isAarch64;
+    broken = stdenv.isAarch64 && stdenv.isLinux;
```

Comment:
Does it really break on Linux? Linux is the primary platform ROOT is used on IMO.